code/modules/holodeck/items.dm

```diff
@@ -26,6 +26,10 @@
 	if(!sword_color_icon)
 		sword_color_icon = pick("red", "blue", "green", "purple")
 
+/obj/item/melee/energy/sword/holographic/make_transformable()
+	. = ..()
+	GetComponent(/datum/component/transforming).clumsy_damage_type = STAMINA
```

Comment:
Pass this as an argument in the component initialization instead of getting and setting it here please.